packages/@sanity/vision/src/components/QueryErrorDialog.styled.js

```diff
@@ -0,0 +1,7 @@
+import {Code} from '@sanity/ui'
+import styled from 'styled-components'
+import {hues} from '@sanity/color'
+
+export const ErrorCode = styled(Code)`
+  color: ${hues.red['700'].hex};
```

Comment:
In order for this to work with the custom theming people might have in the Studio, we need to use the CSS variables:

```suggestion
  color: ${({theme}) => theme.sanity.color.muted.critical.enabled.fg};
```